Ports/indent/package.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash ../.port_include.sh
+port=indent
+version=2.2.11
+files="https://ftp.gnu.org/gnu/indent/indent-${version}.tar.gz indent-${version}.tar.gz
+https://ftp.gnu.org/gnu/indent/indent-${version}.tar.gz.sig indent-${version}.tar.gz.sig
+https://ftp.gnu.org/gnu/gnu-keyring.gpg gnu-keyring.gpg"
+useconfigure=true
+auth_type="sig"
+auth_opts="--keyring ./gnu-keyring.gpg indent-${version}.tar.gz.sig"
+
+man_dir=$SERENITY_ROOT/Build/Root/usr/local/share/man/man1/
+
+post_install() {
+    mkdir -p "${man_dir}"
+    cp "indent-${version}/man/indent.1" "${man_dir}"
```

Comment:
What's the point of installing roff man pages if they're incompatible with our native Markdown manpages?

Reply:
> 
> 
> What's the point of installing roff man pages if they're incompatible with our native Markdown manpages?

It seemed like a nice to have?

Perhaps one day *roff pages will be compatible with the native `man` utility.

Perhaps users will install GNU `man` in `/usr/local/bin/man` so they can benefit from traditional man pages. Other ports already install man pages in `/usr/local/bin/man/man1`.

Reply:
I wonder how hard it would be to parse these, groff was one weird format last time I looked at it...
_/me wonders about it without actually spending the time to check it_

Reply:
> 
> 
> I wonder how hard it would be to parse these, groff was one weird format last time I looked at it...
> _/me wonders about it without actually spending the time to check it_

Yeah I was wondering the same thing. Ideally, during installation of ports, we could convert man pages from *roff to markdown for use with Serenity's `man`. A quick Google search revealed markdown to *roff convertors, but not the reverse. Although I didn't look very hard.

Installing the man pages as is seemed like the next best option in this instance, as I went out of my way to remove generation and installation of man pages and docs from the `Makefile` (to avoid dependency on `texi2html`).

Reply:
We do indeed have the `mandoc` port which can be used to read roffy man pages. I do like the idea of converting man pages to markdown though :)